examples/websub-service-integration-sample/websub_service_integration_sample.metatags

```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates any of the Ballerina's WebSub components (Publisher, Hub, or Subscriber) could be used with non-Ballerina components.
```

Comment:
Edit: This BBE demonstrates how any of Ballerina's WebSub components (Publisher, Hub, or Subscriber) could be used with non-Ballerina components.